pypeit/scripts/coadd_2dspec.py

```diff
@@ -189,6 +190,20 @@ def main(args):
 #            set(np.arange(spectrograph.ndet) + 1) - set(detectors)])))
         msgs.info(f'Detectors to work on: {detectors}')
 
+        # Only_slits?
```

Comment:
only_slits = [int(item) for item in args.only_slits.split(',')] if args.only_slits is not None else None